scripts/cybershake/auto_submit.py

```diff
@@ -113,7 +116,7 @@ def submit_script_to_scheduler(script_name, target_machine=None, **kwargs):
             {
                 "XYTS_PATH": os.path.join(
                     sim_struct.get_lf_outbin_dir(sim_dir),
-                    "{}_xyts.e3d".format(run_name.split("_")[0]),
+                    "{}_xyts.e3d".format(run_name),
```

Comment:
why is split no longer needed?

Reply:
https://github.com/ucgmsim/slurm_gm_workflow/commit/41afab43be6a16d7c5e956092a4a4e64ae284282#diff-23c9da3698676160c79b0fe584cfc923adcc553e412af402d8f1078f9516ff7bL195

Run name (aka e3d filenames) became the realisation name rather than the fault name.